intelmq-manager/js/monitor.js

```diff
@@ -316,6 +329,32 @@ function select_bot(bot_id, history_push = false) {
 }
 }
 
+function refresh_parameters_panel(bot_id) {
+    if (!app.nodes) {
+        // we're not yet ready, buffer the bot for later
+        buffered_bot = [bot_id];
+        return;
+    }
+    let $panel = $("#parameters-panel .panel-body");
+    $panel.text("");
+    var params = app.nodes[bot_id].parameters;
+    console.log(params);
+    for (let key in params) {
+        $el = $("<li><b>"+key+"</b>: "+params[key]+"</li>");
+        console.log(params[key],key);
+        if (params[key].indexOf && params[key].indexOf(ALLOWED_PATH) === 0) {
```

Comment:
In the console I see this error message:
```
TypeError: params[key] is null       monitor.js:345:1
refresh_parameters_panel             http://localhost/js/monitor.js:345:1
select_bot                                     http://localhost/js/monitor.js:320:9
select_bot_func/<                          http://localhost/js/monitor.js:378:21
```
I think it is related to this line.

Reply:
I hope I've corrected this.